packages/rum-core/src/boot/rumPublicApi.ts

```diff
@@ -129,7 +134,7 @@ export function makeRumPublicApi(
     const startRumResults = startRumImpl(
       configuration,
       () => ({
-        user,
+        user: userContextManager.get(),
         context: globalContextManager.get(),
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**❓ question:** ‏ ‏Should we not use those `get` methods or not deprecate them?